src/components/Profile/Profile.styles.ts

```diff
@@ -0,0 +1,56 @@
+import styled from "styled-components";
+
+export const ProfileContainer = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  width: 20rem;
+  padding-top: 1rem;
+  padding-bottom: 1rem;
+`;
```

Comment:
We're using the ProfileContainer inside the Profile component, so we can call it just "Container"

```suggestion
export const Container = styled.div`
  display: flex;
  align-items: center;
  width: fit-content;
  gap: 1rem;
`;
```